metaflow/datastore/local_backend.py

```diff
@@ -215,15 +215,10 @@ def load_bytes(self, paths):
             file_result = None
             metadata = None
             if os.path.exists(full_path):
-                try:
-                    file_result = LazyFile(full_path)
-                except OSError:
-                    pass
-            if file_result:
                 if os.path.exists("%s_meta" % full_path):
                     with open("%s_meta" % full_path, mode='r') as f:
                         metadata = json.load(f)
-                results[path] = (file_result, metadata)
+                results[path] = (full_path, metadata)
```

Comment:
As I mentioned, I would rather return a BufferedIOBase (which is what was being returned). This gives more flexibility to developers of the backends to return something that is not a file. It does not really add much complexity.

Reply:
we can have a separate discussion but I don't think it is worth introducing a dummy object which is not needed for anything currently or in the foreseeable future. We have no idea what new datastores might need in the future (they might require other changes too), so we can easily make the necessary changes later once we know what's actually needed.